quickstarts/microsoft.machinelearningservices/machine-learning-job-create-sweep-job/main.bicep

```diff
@@ -0,0 +1,119 @@
+// Sweep Job Resource using ARM Template
```

Comment:
Do you know anything about this error from Pipelines:

```
##[error]D:\a\1\s\quickstarts\microsoft.machinelearningservices\machine-learning-job-create-sweep-job\main.bicep(1,1) : Warning linter-internal-error: Analyzer 'core/artifacts-parameters' encountered an unexpected exception. Attempted to access a missing method.
```

Reply:
Not sure, tried removing comment as well, also making sure usage defined here: https://docs.microsoft.com/en-us/azure/azure-resource-manager/bicep/linter-rule-artifacts-parameters warning still persists.

Reply:
I think this is a bicep bug, nothing related to the sample...